kernel-commits.mk

```diff
@@ -0,0 +1,7 @@
+KERNEL_COMMIT_amd64_v5.10.186_generic = c23a03114a05
```

Comment:
I understand kernel-commits.mk it's gonna be auto-generated, but since it's seems that's not gonna be generated all the time, is really needed to stay as an additional file? Now we have 2 .mk files only for kernel...

Reply:
@rucoder , as we discussed, this auto-generation will only happen from time to time when kernel is updated, so let's keep both files.